TrkFilters/src/TimeClusterFilter_module.cc

```diff
@@ -77,11 +78,15 @@ namespace mu2e
         // associate to the hit cluster which triggers.  Note there may be other hit clusters which also pass the filter
         // but filtering is by event!
         size_t index = std::distance(tccol->begin(),itc);
-        triginfo->_hitCluster = art::Ptr<TimeCluster>(tcH,index);
+	if (trig_ind < TriggerInfo::MaxNObj){
+	  triginfo->_hitClusters[trig_ind] = art::Ptr<TimeCluster>(tcH,index);
```

Comment:
Is this perhaps left over from the array implementation?   Maybe push_back is needed?